### ISSUE TITLE
Errors running LLAPI example?

### ISSUE THREAD

**Issue by sdobbs** [JeffersonLab/ccdb#37]
I've run into some errors that I hadn't before when using the low-level API.  Maybe the best illustration is using the sample script $CCDB_HOME/examples/llapi_readout.py

I tried running on ifarm1401 with python2.7 using ccdb_1.06.01 and ccdb_1.06.01 and get the following output:

=======================

ifarm1401> python llapi_readout.py
<Directory 3 'test_vars'>
test_vars
/test/test_vars
<Directory 1 'test'>
[]
== TABLE == 'test_table'
/test/test_vars/test_table
Test type
2014-04-10 17:20:28
test_vars
[<ConstantSet '1'>, <ConstantSet '2'>, <ConstantSet '4'>, <ConstantSet '5'>]
x y z
double double double
rows 2 x 3 columns
2
== TABLE == 'test_table2'
/test/test_vars/test_table2
Test type 2
2014-04-10 17:20:28
test_vars
[<ConstantSet '3'>]
c1 c2 c3
int int int
rows 1 x 3 columns
2

== Getting tables another way ==
/test/test_vars/test_table2
test_table
test_table2
test_table
test_table2

== Getting all table data ==
test
0
2147483647
/test/test_vars/test_table2:0:test:2012-09-30_23-48-42
2
[[u'10', u'20', u'30']]
[u'10', u'20', u'30']

== Getting assignment ==
Traceback (most recent call last):
  File "llapi_readout.py", line 91, in <module>
    assignment = provider.get_assignment(1, "/test/test_vars/test_table2", "test")  # run, table, variation
  File "/u/group/halld/Software/builds/Linux_CentOS6-x86_64-gcc4.9.2/ccdb/ccdb_1.06.02/python/ccdb/provider.py", line 1026, in get_assignment
    assert isinstance(path_or_table, TypeTable)
AssertionError

**Comment by markito3** [comment]
We are up to 1.06.02 now, and I think that is what you are using judging from the example given. Not that I think it will make a difference for this issue.

Also see https://halldweb.jlab.org/dist/version_1.28.xml

**Comment by sdobbs** [comment]
For what it's worth, I also tried 1.06.01 and got the same error.  I seem to remember getting some error like this before, but thought it was fixed...

**Comment by DraTeots** [comment]
Could you try 1.07 branch please?

**Comment by sdobbs** [comment]
Same error with 1.07.

**Comment by sdobbs** [comment]
Just wanted to see if there was any new insight on this issue?

**Comment by DraTeots** [comment]
I've fixed it. Look in master.

The right get_assignment signature is 

``` python
get_assignment(table_path, run, variation, date=None)
```

While in examples run was the first argument and table_path the second. 
The root of the problem is that at some point arguments order was changed to be consistent with the order of tokens of
requests `path:run:version:date`. 

While unit tests are run on commits, examples are not run automatically. So seems that it just hasn't beet run since the refactoring.
